internal/bminventory/inventory.go

```diff
@@ -376,7 +376,7 @@ func getDefaultNetworkType(params installer.V2RegisterClusterParams) (*string, e
 		return params.NewClusterParams.NetworkType, nil
 	}
 
-	isOpenShiftVersionRecentEnough, err := common.BaseVersionGreaterOrEqual(swag.StringValue(params.NewClusterParams.OpenshiftVersion), minimalOpenShiftVersionForDefaultNetworkTypeOVNKubernetes)
+	isOpenShiftVersionRecentEnough, err := common.BaseVersionGreaterOrEqual(minimalOpenShiftVersionForDefaultNetworkTypeOVNKubernetes, swag.StringValue(params.NewClusterParams.OpenshiftVersion))
```

Comment:
why did you switch the order of the parameters?

Reply:
logically it was the opposite. I noticed that during that PR and updated it.
It should check if `params.NewClusterParams.OpenshiftVersion` >= `minimalOpenShiftVersionForDefaultNetworkTypeOVNKubernetes`.